app/controllers/admin/categories_controller.rb

```diff
@@ -57,18 +57,22 @@ def update
   end
 
   def order
-    sort_priorities = params[:order].each_with_index.map do |category_id, index|
+    sort_priorities = params[:order]
+                      .reject { |o| !o.match /[0-9]+/} #Guard against sql injection
```

Comment:
this probably isn't needed, since the ID:s are matched to actual category ID:s later on, and not entered into any SQL query before that